ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/CoveragesToggleGroup.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useContext } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import { ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
+
+import { complianceEnhancedCoveragePath } from 'routePaths';
+
+import { ComplianceProfilesContext } from './ComplianceProfilesProvider';
+
+function CoveragesToggleGroup({ tableView = 'checks' }: { tableView: string }) {
+    const { profileName } = useParams();
+    const history = useHistory();
+
+    const { profileScanStats } = useContext(ComplianceProfilesContext);
+
+    const handleToggleChange = (selectedProfile) => {
+        history.push(`${complianceEnhancedCoveragePath}/profiles/${selectedProfile}/${tableView}`);
+    };
+    return (
+        <ToggleGroup aria-label="Toggle for selected profile view">
+            {profileScanStats &&
+                profileScanStats.scanStats.map((profile) => (
```

Comment:
Is optional chaining `profileScanStats?.scanStats.map((profile) => (…))` an alternative?

What are pro and con:
* Add `TODO` comments wherever this temporary data structure is used.
* Find in files, when the time comes. Hopefully sooner than later.

Reply:
Updated

Reply:
Assuming we'll be changing the context value type, typescript should complain everywhere this is being used.